PVM_PVP/Any/ModernSlayer.py

```
@@ -0,0 +1,493 @@
+            return False
+        is_valid_item = is_spellbook(item.Name) if skill == "Magery" else skill_lower in props
+        return is_valid_item
+    return is_valid
+
+def find_all_valid_slayer_items(container, slayer_filter, slayer_list=None):
+    """Recursively find all valid slayer items in containers."""
+    if slayer_list is None:
+        slayer_list = []
+    try:
+        contents = API.ItemsInContainer(container.Serial)
+    except:
```

Ruff (0.13.1)

Do not use bare `except`

(E722)